### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by jajajajaja121** [lucastabelini/LaneATT#15]
I have run your test code in my computer, and the environment is below,
```
absl-py                   0.11.0                    <pip>
addict                    2.3.0                     <pip>
albumentations            0.5.0                     <pip>
appdirs                   1.4.4                     <pip>
argon2-cffi               20.1.0                    <pip>
async_generator           1.10                       py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
asynctest                 0.13.0                    <pip>
attrs                     20.3.0             pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
attrs                     20.2.0                    <pip>
backcall                  0.2.0                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bleach                    3.2.1                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ca-certificates           2020.10.14                    0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cachetools                4.2.0                     <pip>
certifi                   2020.6.20          pyhd3eb1b0_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cffi                      1.14.3                    <pip>
chardet                   3.0.4                     <pip>
cityscapesScripts         2.1.7                     <pip>
click                     7.1.2                     <pip>
codecov                   2.1.10                    <pip>
colorama                  0.4.4                     <pip>
coloredlogs               14.0                      <pip>
coverage                  5.3                       <pip>
cudatoolkit               8.0                           3    <unknown>
cycler                    0.10.0                    <pip>
Cython                    0.29.21                   <pip>
decorator                 4.4.2                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
defusedxml                0.6.0                     <pip>
defusedxml                0.6.0                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
dill                      0.3.3                     <pip>
entrypoints               0.3                       <pip>
entrypoints               0.3                      py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
filelock                  3.0.12                    <pip>
flake8                    3.8.4                     <pip>
future                    0.18.2                    <pip>
gdown                     3.12.2                    <pip>
google-auth               1.24.0                    <pip>
google-auth-oauthlib      0.4.2                     <pip>
grpcio                    1.34.0                    <pip>
humanfriendly             8.2                       <pip>
idna                      2.10                      <pip>
imagecorruptions          1.1.0                     <pip>
imageio                   2.9.0                     <pip>
imgaug                    0.4.0                     <pip>
importlib-metadata        2.0.0                      py_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
importlib_metadata        2.0.0                         1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
iniconfig                 1.1.1                     <pip>
interrogate               1.3.1                     <pip>
ipykernel                 5.3.4            py36h5ca1d4c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipython                   7.16.1           py36h5ca1d4c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipython-genutils          0.2.0                     <pip>
ipython_genutils          0.2.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipywidgets                7.5.1                     <pip>
isort                     4.3.21                    <pip>
jedi                      0.17.2                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jinja2                    2.11.2                     py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
joblib                    1.0.0                     <pip>
jsonschema                3.2.0                      py_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyter                   1.0.0                     <pip>
jupyter-console           6.2.0                     <pip>
jupyter_client            6.1.7                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyter_core              4.6.3                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyterlab_pygments       0.1.2                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
kiwisolver                1.2.0                     <pip>
kwarray                   0.5.9                     <pip>
libedit                   3.1.20191231         h14c3975_1    defaults
libffi                    3.2.1             hf484d3e_1007    defaults
libgcc-ng                 9.1.0                hdf63c60_0    defaults
libsodium                 1.0.18               h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libstdcxx-ng              9.1.0                hdf63c60_0    defaults
lxml                      4.5.0                     <pip>
Markdown                  3.3.3                     <pip>
markupsafe                1.1.1            py36h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
matplotlib                3.1.3                     <pip>
mccabe                    0.6.1                     <pip>
mistune                   0.8.4            py36h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mmcv                      1.1.5                     <pip>
mmcv-full                 1.1.5                     <pip>
mmdet                     2.7.0                     <pip>
mmlvis                    10.5.3                    <pip>
mmpycocotools             12.0.3                    <pip>
mmsegmentation            0.8.0                     <pip>
multiprocess              0.70.11.1                 <pip>
nb_conda_kernels          2.3.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nbclient                  0.5.1                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nbconvert                 6.0.7                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nbformat                  5.0.8                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ncurses                   6.2                  he6710b0_1    defaults
nest-asyncio              1.4.3                     <pip>
nest-asyncio              1.4.2              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
networkx                  2.5                       <pip>
nms                       0.0.0                     <pip>
notebook                  6.0.3                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
notebook                  6.1.5                     <pip>
numpy                     1.19.2                    <pip>
oauthlib                  3.1.0                     <pip>
opencv-python             4.2.0.32                  <pip>
opencv-python-headless    4.4.0.44                  <pip>
openssl                   1.1.1h               h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ordered-set               4.0.2                     <pip>
p-tqdm                    1.3.3                     <pip>
packaging                 20.4                       py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pandas                    1.1.3                     <pip>
pandoc                    2.11                 hb0f4dca_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pandocfilters             1.4.3            py36h06a4308_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
parso                     0.7.0                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
parso                     0.7.1                     <pip>
pathos                    0.2.7                     <pip>
pexpect                   4.8.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pickleshare               0.7.5                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pickleshare               0.7.5                     <pip>
Pillow                    6.2.2                     <pip>
pip                       20.2.4                   py36_0    defaults
pluggy                    0.13.1                    <pip>
pox                       0.2.9                     <pip>
ppft                      1.6.6.3                   <pip>
prometheus_client         0.8.0                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
prompt-toolkit            3.0.8                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
protobuf                  3.14.0                    <pip>
ptyprocess                0.6.0                     <pip>
ptyprocess                0.6.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
py                        1.9.0                     <pip>
pyasn1                    0.4.8                     <pip>
pyasn1-modules            0.2.8                     <pip>
pycodestyle               2.6.0                     <pip>
pycparser                 2.20                      <pip>
pydensecrf                1.0rc3                    <pip>
pyflakes                  2.2.0                     <pip>
pygments                  2.7.2              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyparsing                 2.4.7                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyquaternion              0.9.9                     <pip>
pyrsistent                0.17.3           py36h7b6447c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytest                    6.1.1                     <pip>
python                    3.6.8                h0371630_0    defaults
python-dateutil           2.8.1                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytz                      2020.1                    <pip>
PyWavelets                1.1.1                     <pip>
PyYAML                    5.3.1                     <pip>
pyzmq                     19.0.2           py36he6710b0_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyzmq                     19.0.2                    <pip>
qtconsole                 4.7.7                     <pip>
QtPy                      1.9.0                     <pip>
readline                  7.0                  h7b6447c_5    defaults
requests                  2.24.0                    <pip>
requests-oauthlib         1.3.0                     <pip>
rsa                       4.6                       <pip>
scikit-image              0.17.2                    <pip>
scikit-learn              0.23.2                    <pip>
scipy                     1.4.1                     <pip>
send2trash                1.5.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
Send2Trash                1.5.0                     <pip>
setuptools                50.3.0           py36hb0f4dca_1    defaults
Shapely                   1.7.0                     <pip>
six                       1.15.0           py36h06a4308_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
six                       1.15.0                    <pip>
sqlite                    3.33.0               h62c20be_0    defaults
tabulate                  0.8.7                     <pip>
tensorboard               2.3.0                     <pip>
tensorboard-plugin-wit    1.7.0                     <pip>
terminado                 0.9.1                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
terminado                 0.9.1                     <pip>
testpath                  0.4.4                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
thop-0.0.31               2005241907                <pip>
threadpoolctl             2.1.0                     <pip>
tifffile                  2020.9.3                  <pip>
tk                        8.6.10               hbc83047_0    defaults
toml                      0.10.1                    <pip>
torch                     1.6.0                     <pip>
torchvision               0.7.0                     <pip>
tornado                   6.0.4            py36h7b6447c_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tornado                   6.1                       <pip>
tqdm                      4.43.0                    <pip>
traitlets                 4.3.3                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
typing                    3.7.4.3                   <pip>
ubelt                     0.9.3                     <pip>
ujson                     1.35                      <pip>
urllib3                   1.25.11                   <pip>
wcwidth                   0.2.5                      py_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
webencodings              0.5.1                     <pip>
webencodings              0.5.1                    py36_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
Werkzeug                  1.0.1                     <pip>
wheel                     0.35.1                     py_0    defaults
widgetsnbextension        3.5.1                     <pip>
xdoctest                  0.15.0                    <pip>
xmljson                   0.2.0                     <pip>
xz                        5.2.5                h7b6447c_0    defaults
yapf                      0.30.0                    <pip>
zeromq                    4.3.3                he6710b0_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zipp                      3.4.0              pyhd3eb1b0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zlib                      1.2.11               h7b6447c_3    defaults
```
basically, it is same with the environment that you list in your requirement.txt, it is a strange error, that I don't know why it is happen, if your know, please help me, Thank you very much!!!

**Comment by jajajajaja121** [comment]
I find it happens when load state dict.

**Comment by lucastabelini** [comment]
You're supposed to use CUDA 10.2 with PyTorch 1.6.

**Comment by jajajajaja121** [comment]
> You're supposed to use CUDA 10.2 with PyTorch 1.6.

OK, thank you.

**Comment by jajajajaja121** [comment]
I have tried pytorch==1.6 and cuda==10.2, but this error is still exist, when load statedict.

**Comment by jajajajaja121** [comment]
I have solved this problem by upgrade python to 3.8
